luigi/scheduler.py

```diff
@@ -878,6 +887,10 @@ def disable_worker(self, worker):
         self._state.disable_workers({worker})
 
     @rpc_method()
+    def set_worker_processes(self, worker, n, diff):
```

Comment:
I left it as `worker` for the moment as all the rpc methods above that one also use `worker`. But in general, this would really clarify the code.